values/jobs/autoconfig.gotmpl

```diff
@@ -0,0 +1,27 @@
+{{- $v := .Values }}
+{{- $c := $v.charts }}
+{{- $k := $c.keycloak }}
+type: Job
+name: autoconfig
+
+image:
+  repository: docker.io/otomi/otomi-e2e
+  tag: autoconfig
+  pullPolicy: Always
+
+script: | 
+    echo "AUTOCONFIG TASK: Creating Keycloak user"
```

Comment:
Users watching autoconfig job already know the context. The perhaps you can remove `AUTOCONFIG TASK: ` prefix.

Reply:
Done